var/spack/repos/builtin/packages/vtr/package.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+from os.path import join as pjoin
+
+from spack.package import *
+
+
+class Vtr(MakefilePackage):
```

Comment:
Looking at https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/master/Makefile I see that the Makefile is meant to be a convenience wrapper. I think we should be using  `CMakePackage` instead of  `MakefilePackage`.

Reply:
The Makefile is indeed a wrapper. However, since `make` is the main build flow as described in the [documents](https://docs.verilogtorouting.org/en/latest/BUILDING/), should we continue using `MakefilePackage` as it is?

Furthermore, upon careful review of this Makefile, I noticed that there are some build types that we can add as variants. What are your thoughts on this?

```python
variant(
    "build_type", default="release", description="Build type for VTR",
    values=("release_pgo", "release", "debug", "strict"), multi=False
)
```
```python
make("BUILD_TYPE={0}".format(spec.variants["build_type"].value))
```